src/main/resources/defaults/data/Banners.json

```diff
@@ -34,49 +34,33 @@
 		"gachaType": 301,
 		"scheduleId": 903,
 		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A097",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A097_TITLE",
+		"prefabPath": "GachaShowPanel_A103",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A081_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1071],
+		"rateUpItems1": [1073],
+		"rateUpItems2": [1073]
 		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
-	},
-	{
-		"comment": "Character Event Banner 2",
-		"gachaType": 400,
-		"scheduleId": 923,
-		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A098",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A036_TITLE",
-		"costItemId": 223,
-		"beginTime": 0,
-		"endTime": 1924992000,
-		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1022],
-		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
+		"weights5":[[1,75],[73,150],[90,10000]],
```

Comment:
Why are you deleting `Character Event Banner 2` and changing the weights to match Standard?